globee/migrations/0004_auto_20181117_2203.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.0.2 on 2018-11-17 22:03
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('globee', '0003_auto_20181117_2107'),
```

Comment:
This migration file doesn't exists.
I guess it would be better if you remove both migration files and run `makemigrations` again to create only one file

Reply:
Oh, thanks. There's an incorrect migration filename in dependencies. I'll squash them into one anyway.